packages/widget/tests/unit/components/resource-amount-selector/resource-amount-selector.test.ts

```diff
@@ -221,6 +233,8 @@ describe('Resource amount selector component - sygma-resource-amount-selector',
       input.dispatchEvent(new Event('input'));
       await nextFrame();
 
+      await aTimeout(600);
```

Comment:
it would be nice to create a const INPUT_DEBOUNCE_TIME = 600 to make sure that it's not a magic number :)